Java2020/src/lesson201222/phonecalls/CallLog.java

```diff
@@ -0,0 +1,23 @@
+package lesson201222.phonecalls;
+
+import java.time.LocalDateTime;
+
+public class CallLog {
+
+	
+	LocalDateTime callDateTime;
+	
+	String number;
+	
+	String callType;
+	
+	public CallLog(LocalDateTime callDateTime, String number, String callType) {
```

Comment:
It's not a log, it's just one item of it. Better name CallLogItem or CallLogRecord